src/App.jsx

```diff
@@ -0,0 +1,234 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import './App.css';
+import { Goods } from './components';
+
+const goodsFromServer = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+const defaultLengthlimit = 1;
+
+function Start({ runAplication }) {
```

Comment:
```suggestion
function startApp({ runAplication }) {
```
называй функции с маленкой буквы